repository/u.json

```diff
@@ -22,11 +22,15 @@
 			]
 		},
 		{
-			"details": "https://github.com/frankban/UbuntuPaste",
+			"details": "https://github.com/3v1n0/UbuntuPaste",
 			"releases": [
 				{
 					"sublime_text": "<3000",
-					"branch": "master"
+					"tags": "2."
```

Comment:
This tag setup won't work because after stripping the prefix specified here (`2.`), the remaining tag name is not a valid semver anymore (only two parts).